scripts/update-spectrum-css.js

```diff
@@ -65,3 +72,21 @@ async function update() {
 }
 
 update();
+
+async function getLatestVersion(packageName, currentVersion) {
+    const targetVersion = await latestVersion(packageName);
+    const same = currentVersion.replace('^', '') === targetVersion;
+    return same ? undefined : `^${targetVersion}`;
+}
+
+async function getSemverVersion(packageName, currentVersion) {
+    const allVersions = await new Promise((resolve, reject) => {
+        pkgVersions(packageName, (error, versions) => {
+            if (error) return reject(error);
+            resolve(versions);
+        });
+    });
```

Comment:
Is it possible to wrap this in a try/catch so that we're not mixing async/await with new Promise? Maybe leveraging the `version` options in `latestVersion` can help normalize that here? https://github.com/sindresorhus/package-json#version